tsl/test/isolation/specs/CMakeLists.txt

```diff
@@ -15,27 +15,21 @@ list(
   cagg_insert.spec
   cagg_multi_iso.spec
   cagg_concurrent_refresh.spec
-  cagg_concurrent_refresh_dist_ht.spec
   deadlock_drop_chunks_compress.spec)
 
+if(ENABLE_MULTINODE_TESTS)
```

Comment:
Should it also match DEBUG build only here and below? if (`ENABLE_MULTINODE_TESTS AND CMAKE_BUILD_TYPE MATCHES Debug`) ... Or maybe invalidate `ENABLE_MULTINODE_TESTS` globally if not in DEBUG build

Reply:
This test didnt require DEBUG previously and the tests below are nested in a DEBUG check

Reply:
Not all multinode tests require DEBUG and I kept these requirements intact.